### ISSUE TITLE
What --cpus CPUS means

### ISSUE THREAD

**Issue by xiekunwhy** [weberlab-hhu/Helixer#152]
**Describe the bug**
--cpus option in HybridModel.py seems not work

**To Reproduce**
```
singularity run /public2/home/sl_qybio/sl_qybio/software/biosoft/HelixerDocker/helixer.sif HybridModel.py --overlap --val-test-batch-size 32 -v --cpus 4 --prediction-output-path /public2/home/sl_qybio/sl_qybio/project/2024/QY240516GA057_LanHua_genome/QY240516GA057_std_3_annot/pipe/05.abinitio/helixer/02.predict/Chr01-1-10000000.predict.h5 --load-model-path /public2/home/sl_qybio/sl_qybio/software/biosoft/Helixer/models/land_plant_v0.3_a_0080.h5 --test-data /public2/home/sl_qybio/sl_qybio/project/2024/QY240516GA057_LanHua_genome/QY240516GA057_std_3_annot/pipe/05.abinitio/helixer/01.fasta2h5/Chr01-1-10000000.h5
```

**Error**
--cpus 4 , use not only 4 cpus but over 60 cpus
![image](https://github.com/user-attachments/assets/9440860f-ae4a-40cd-8a9b-8e0499cb482f)

**Environment (please complete the following information):**
 - OS: linux
 - singularity
 - if the web tool at https://www.plabipd.de/helixer_main.html was used: no

**Comment by felicitas215** [comment]
Hi,
thank you for your message. First of, yes `--cpus` is deprecated. It doesn't appear in the Helixer options documentation (https://github.com/weberlab-hhu/Helixer/blob/main/docs/helixer_options.md) and in the arguments list when using `-h`/`--help` in version 0.3.4. If you want to set multiple CPU cores to fetch the input data, you would need to use `--workers`. This is however just for training the model, for prediction it is not necessary/implemented. Second, you didn't set `singularity run --nv ...` for a process that is recommended to and can use the GPU, your system is using the CPU cores to predict, resulting in the high CPU usage.

**Comment by felicitas215** [comment]
I'm sorry, we apparently forgot to suppress this argument when using the help function. This will be fixed soon.

**Comment by felicitas215** [comment]
Fixed in release [v0.3.4](https://github.com/weberlab-hhu/Helixer/tree/v0.3.4), using `--cpus` will result in a deprecation warning and will be ignored.
